### PR TITLE
Contribution guidelines

### PR DESCRIPTION
**Pull request by dennisguse**

### CODE REVIEW

**Review by rgmf — APPROVED** [review]
:ok_hand:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Testing 
+
+Just use OpenTracks and tell us what can be improved by opening an issue [here](https://github.com/OpenTracksApp/OpenTracks/issues/new/choose)
+
+If we broke things (happens now and then), please tell us also.
+
+## Coding new features
+
+You have cool idea for a feature that is missing in OpenTracks?
```
Comment (pstorch) [inline CONTRIBUTING.md:11]: have **a** cool

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+And you would like to help by implementing it?
+If not, you can just take a look at the [issue tracker](https://github.com/OpenTracksApp/OpenTracks/issues).
+
+Helpful hints:
+* creating an issue beforehand might be helpful
+  Sometimes it is helpful to discuss your idea and also your implementation approach first.
```
Comment (pstorch) [inline CONTRIBUTING.md:17]: Three times "helpful" in a row. 
Maybe "Tips" or "Advice" as the headline and "... might be good" in the second line.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ 
+## Coding conventions
+
+None so far.
+Just keep it simple.
+And follow the _scout rule_. 
+
+You can take a look at existing code and get a feeling for it.
+
+Please autoformat your using AndroidStudio's code formatter.
```
Comment (pstorch) [inline CONTRIBUTING.md:35]: ... autoformat your **code** using ...

**Comment by pstorch** [comment]
Was too late, sorry

**Pull request merged**